docs/monitors/interface.md

```diff
@@ -0,0 +1,69 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# interface
```

Comment:
This name could be improved.  Maybe `net-io` or something.

Reply:
ack